api/sql/schema/750$rule.decision.fetch.sql

```diff
@@ -123,6 +130,22 @@ BEGIN
         c.priority,
         l.limitId
 
+    IF @isTransactionValidate = 0 AND @isMaskChecked = 1 AND @sourceCheckAmount <> 0 AND @sourceCheckAmount <= @amount
```

Comment:
This code should be changed to return limit data in case of validation.
Amount checks should not be done here, as they happen in the code below.